collectors/ebpf.plugin/README.md

```diff
@@ -191,6 +191,50 @@ The eBPF collector enables and runs the following eBPF programs by default:
 -   `network viewer`: This eBPF program creates charts with information about `TCP` and `UDP` functions, including the
     bandwidth consumed by each.
 
+### `[network viewer]`
+
+You can configure the information shown on `outbound` and `inbound` charts with the settings in this section. 
+
+```conf
+[network viewer]
+    maximum dimensions = 500
+    ports = 1-1024 !145 !domain
+    hostnames = !example.com
+    ips = !127.0.0.1/8 10.0.0.0/8 172.16.0.0/12 192.168.0.0/16 fc00::/7
+```
+
+When you define a `ports` setting, Netdata will collect network metrics for that specific port. For example, if you
+write `ports = 19999`, Netdata will collect only connections for itself. The `ports`, `hostnames`, and `ips` settings
+all accept [simple patterns](/libnetdata/simple_pattern/README.md).
```

Comment:
IPs and ports don't support simple patterns and they shouldn't. But we need to state it here somehow.

Reply:
I updated it, thanks to call attention